rotkehlchen/tests/unit/decoders/test_aerodrome.py

```diff
@@ -311,8 +311,8 @@ def test_unlock_aero(base_accounts, base_transaction_decoder):
             sequence_index=140,
             timestamp=timestamp,
             location=Location.BASE,
-            event_type=HistoryEventType.SPEND,
-            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            event_type=HistoryEventType.INFORMATIONAL,
```

Comment:
Why make the burning informational? That means the NFT won't be trackable anymore